.github/workflows/deploy-nginx-staging.yaml

```diff
@@ -7,111 +7,26 @@ on:
-          pipx install --pip-args=--constraint=.github/workflows/constraints.txt poetry
-          pipx install --pip-args=--constraint=.github/workflows/constraints.txt nox
-          pipx inject --pip-args=--constraint=.github/workflows/constraints.txt nox nox-poetry
-
-      - name: Run contract-tests with nox
-        run: nox -s contract_tests
-        env:
-          API_KEY: ${{ secrets.API_KEY }}
-
-      - name: Build and push
-        id: docker_build
-        uses: docker/build-push-action@v2
-        with:
-          context: ./nginx/
-          file: ./nginx/Dockerfile
-          push: true
-          tags: ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels }}
-          cache-from: type=local,src=.cache
-          cache-to: type=local,dest=.cache
-          build-args: |
-            GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}
-
-      - name: Apply Helm template
-        run: |
-          helm repo add fdk https://informasjonsforvaltning.github.io/helm-chart/
-          helm fetch --untar --untardir ./helm fdk/static-rdf-nginx
-          helm template --set DOCKER_IMAGE_NAME="eu.gcr.io/digdir-fdk-infra/static-rdf-nginx:${{ steps.meta.outputs.version }}" \
-          -f ./helm/static-rdf-nginx/values/staging.yaml \
-          ./helm/static-rdf-nginx/ > ./kubectlapply.yaml
-          chmod o+w ./kubectlapply.yaml
-          cat kubectlapply.yaml
-
-      - name: Setup gcloud CLI for deploy
-        uses: google-github-actions/auth@v0
-        with:
-          credentials_json: ${{ secrets.DIGDIR_FDK_DEV_AUTODEPLOY }}
-
-      - name: Deploy to GCP & check status
-        run: |
-          gcloud container clusters get-credentials digdir-fdk-dev --region europe-north1-a --project digdir-fdk-dev
-          kubectl apply -f ./kubectlapply.yaml --force
-          if ! kubectl --namespace=staging rollout status deployment static-rdf-nginx; then
-            kubectl --namespace=staging rollout undo deployment static-rdf-nginx
-            kubectl --namespace=staging rollout status deployment static-rdf-nginx
-            exit 1
-          fi
-
-      - name: Notify slack
-        uses: 8398a7/action-slack@v3
-        with:
-          status: ${{ job.status }}
-          author_name: "Application: static-rdf-nginx | Environment: staging"
-          fields: commit,author,workflow
-        env:
-          GITHUB_TOKEN: ${{ github.token }}
-          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
-        if: always()
+    uses: Informasjonsforvaltning/workflows/.github/workflows/build-deploy-nox.yaml@main
+    with:
+      app_name: static-rdf-nginx
+      caller_sha: ${{ github.sha }}
+      repo: ${{ github.repository }}
+      actor: ${{ github.actor }}
+      python_version: '3.9'
+      python_architecture: x64
+      environment: staging
+      cluster: digdir-fdk-dev
+      nox_secret: True
+      nox_cmd: '-s contract_tests'
+      dockerfile_context: ./nginx/
```

Comment:
Eg trur at dette er feil og årsak til at bygget tryner. Det virkar som den leitar etter ./nginx/nginx/blabla.conf

Det er ein sub-folder for mykje. .conf  og dockerfile ligg under ./nginx-mappa: 
```
nginx/Dockerfile
nginx/nginx.conf
```